### PR TITLE
mod_tile: Update to version v0.7.2

### PR DESCRIPTION
**Pull request by frankdean**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 14.7.2 23H311 arm64
Xcode 16.2 16C5032a

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: gis/mod_tile/Portfile [hunk 1]
```diff
@@ -5,12 +5,10 @@ PortGroup           active_variants 1.1
 PortGroup           boost 1.0
 PortGroup           github 1.0
 
-github.setup        openstreetmap mod_tile 0.5
+github.setup        openstreetmap mod_tile v0.7.2
```
Comment (reneeotten) [inline gis/mod_tile/Portfile:8]: this doesn't look right, the "v" shouldn't be part of the "version". It should likely be:
".... mod_tile 0.72 v"

**Review by reneeotten — CHANGES_REQUESTED** [review]

**Comment by reneeotten** [comment]
please check the failure(s) on the CI

File: gis/mod_tile/Portfile (hunk 1 above)
Comment (frankdean) [reply gis/mod_tile/Portfile:8]: I was assuming the version needs to match a [tag/release](https://github.com/openstreetmap/mod_tile/tags).  It's certainly some variation around `0.7.2` but that failed to fetch a release.

**Comment by frankdean** [comment]
> please check the failure(s) on the CI

I'm trying to figure out why `autoreconf` is failing on CI which is painful when it works fine for me on `macos-14`.  I've fixed all the obsolete macro warnings, but it still fails apparently on a `PKG_CHECK_MODULES` entry.  Do you know what version of `autoreconf` is running on macos-14?

**Comment by kencu** [comment]
> > please check the failure(s) on the CI
> 
> I'm trying to figure out why `autoreconf` is failing on CI which is painful when it works fine for me on `macos-14`. I've fixed all the obsolete macro warnings, but it still fails apparently on a `PKG_CHECK_MODULES` entry. Do you know what version of `autoreconf` is running on macos-14?

most likely is missing pkgconfig build dep

**Comment by frankdean** [comment]
> > > please check the failure(s) on the CI
> > 
> > 
> > I'm trying to figure out why `autoreconf` is failing on CI which is painful when it works fine for me on `macos-14`. I've fixed all the obsolete macro warnings, but it still fails apparently on a `PKG_CHECK_MODULES` entry. Do you know what version of `autoreconf` is running on macos-14?
> 
> most likely is missing pkgconfig build dep

Thank you so much!  It does seem that was it.  Thanks again.

File: gis/mod_tile/Portfile [hunk 2]
```diff
@@ -120,9 +115,9 @@ if {![variant_isset postgresql15] && ![variant_isset postgresql16]} {
     default_variants-append +postgresql16
 }
 
-post-extract {
-    file rename ${worksrcpath}/renderd.conf ${worksrcpath}/renderd.conf.orig
-}
+# post-extract {
+#     file copy ${worksrcpath}/etc/renderd/renderd.conf ${worksrcpath}/etc/renderd/renderd.conf.orig
+# }
```
Comment (reneeotten) [inline gis/mod_tile/Portfile:120]: don't comment lines out; if it's not needed anymore just remove

File: gis/mod_tile/Portfile [hunk 3]
```diff
@@ -187,7 +182,7 @@ post-destroot {
         ${destroot}${docdir}/README_MacPorts.md
 
     xinstall -m 0755 -d ${destroot}${prefix}/etc/renderd
-    file rename ${destroot}${prefix}/etc/renderd.conf.orig ${destroot}${prefix}/etc/renderd/renderd.conf.orig
+    # file rename ${destroot}${prefix}/etc/renderd/renderd.conf.orig ${destroot}${prefix}/etc/renderd/renderd.conf.orig
```
Comment (reneeotten) [inline gis/mod_tile/Portfile:185]: remove if no longer needed

File: gis/mod_tile/Portfile (hunk 2 above)
Comment (frankdean) [reply gis/mod_tile/Portfile]: Removed

File: gis/mod_tile/Portfile (hunk 3 above)
Comment (frankdean) [reply gis/mod_tile/Portfile]: Removed

**Comment by frankdean** [comment]
@reneeotten Thanks!

**Pull request merged**